pumble-sdk/src/api/v1/MessagesApiClientV1.ts

```diff
@@ -183,6 +184,19 @@ export class MessagesApiClientV1 extends BaseApiClient {
         });
     }
 
+    public async deleteEphemeralMessage(messageId: string, channelId: string, payload: V1.DeleteEphemeralMessageRequestBody): Promise<void> {
+        if (payload.deleteForUsers.length <= 0) {
+            throw new Error("The list of users to whom the message should be deleted is empty.")
+        }
+
+        const url = this.urls.deleteEphemeralMessage(channelId, messageId);
+        return await this.request({
+            method: 'POST',
```

Comment:
Shouldn't this be a DELETE ? 

I made the comment in the backend PR as well.

Reply:
Updated.